packages/service-library/src/servicelib/fastapi/http_client_thin.py

```diff
@@ -203,7 +203,7 @@ def __init__(
         base_url: URLTypes | None = None,
         default_http_client_timeout: TimeoutTypes | None = None,
         extra_allowed_method_names: set[str] | None = None,
-        tracing_settings: TracingSettings | None,
+        tracing_settings: TracingSettings | None = None,
```

Comment:
why is that one necessary? no tracing here?

Reply:
Never mind me, got confused by the position. We do want tracing all over the place.